website/src/app/product/early-access/page.tsx

```diff
@@ -59,6 +59,29 @@ export default function EarlyAccess() {
           </div>
         </div>
       </div>
+      {/*
+          Feature section 1: Flexible, zero-trust access controls, as granular as you need them
+
+Looking to replace your VPN with a zero-trust approach? With Firezone 1.0 you have the flexibility to implement zero-trust access according to your organization's needs. Protect everything from an entire subnet all the way down to individual applications, and everything in between. Firezone intelligently routes allowed traffic to protected resources, leaving them invisible to unauthorized outsiders.
```

Comment:
I think we should say "Firezone 1.0" in every single paragraph, instead, we can just say that all early access features are released as part of V1.0.

Reply:
👍